extra_dist/install-fq-mods.ps1

```diff
@@ -0,0 +1,127 @@
+    $webClient = New-Object System.Net.WebClient
+    try {
+        $webClient.DownloadFile($strFileURL, $strHDLocation)
+        Write-Host "`n >> Perl v $strVersion was downloaded OK <<`n"
+    }
+    catch {
+        Write-Host "`nFailed to download Perl Install File. Unable to Continue`n"
+        Pause
+    }
+    Write-Host "`n * Running automated Perl install`n"
+    # Install Perl
+    try {
+        Start-Process msiexec.exe -ArgumentList " /i ""$strHDLocation"" /qb /L*V ""$strLogLogation"" PERL_PATH=Yes PERL_EXT=Yes" -Wait    
+    }
+    catch {
+        Write-Host "`n  Failed to install perl from $env:TEMP\Perl.msi`n"
+        Remove-Item "$env:TEMP\Perl.msi"
+    }
+    
+    Remove-Item "$env:TEMP\Perl.msi"
+    
+    Write-Host "`n >> Perl Install completed <<`n"
+
+    # Strawberry Perl will set the system/machine PATH during install
+    # We need to set the local process PATH variable to finish install of FQ.
+    $env:Path = [System.Environment]::GetEnvironmentVariable('PATH', "Machine")
+}
+
+# Check Perl Version
+function chkPerlVer {
+    
+    Write-Host "`n * Checking Perl version`n"
+   
+    perl -e "exit(int($]));"
+    $perlMajor = $LastExitCode
+    perl -e "`$ver=1000*sprintf('%0.3f', $]); exit(int(`$ver)-5000);"
+    $perlMinor = $LastExitCode
+
+    if ($perlMajor -eq 5) {
+        if ($perlminor -lt $perlVersion) {
+            Write-Host "`n Found perl version $perlMajor.$perlMinor, but GnuCash requires at least version 5.$perlVersion."
+            Write-Host "Please manually install version 5.8 or above of"
+            Write-Host "* ActivePerl (http://www.activestate.com/store/activeperl) or"
+            Write-Host "* Strawberry Perl (http://code.google.com/p/strawberry-perl/downloads/)"
+            Write-Host "and add the bin directory to your Path environment variable. `n"
+            Pause
+            Throw
+        }
+    }
+}
+
+# Function to install the Finance::Quote Module
+function install_fq_mod {
+    Write-Host "`n * Installing required perl modules`n"
+    
+    try {
+        perl -w gnc-fq-update  
+    }
+    catch {
+        "Perl appears to be installed, but cannot install Finance::Quote module"
+        Pause
+    } 
+    
+    Write-Host "  ***"
+    Write-Host "  *** You need an API key (from https://www.alphavantage.co)"
+    Write-Host "  ***   to run the Perl module Finance::Quote."
+    Write-Host "  ***"
+    Write-Host "  *** Make it available to GnuCash by"
+    if ($env:ALPHAVANTAGE_API_KEY) {
+        $done = "(done) "
+    }
+    else {
+        $done = ""
+    }
+    Write-Host "  ***    - setting the environment variable ALPHAVANTAGE_API_KEY $done or"
+    Write-Host "  ***    - starting GnuCash and adding the Alpha Vantage api key in"
+    Write-Host "  ***        Edit->Preferences->Online Quotes"
+    Write-Host "  ***`n"
+    pause
```

Comment:
New item, I should have noticed this before.  First it doesn't really make sense to test `ALPHAVANTAGE_API_KEY` in the middle, but more significantly it's not necessary to set the environment variable at all. Users can set a preference in GnuCash if they need Alphavantage, but that need is much less than it was. I'm inclined towards removing this and leaving it for GnuCash to deal with at runtime.